actors/verifreg/src/state.rs

```diff
@@ -27,11 +27,11 @@ pub struct State {
     // Address of the data cap token actor
     pub token: Address,
     // Maps verifier addresses to data cap minting allowance (in bytes).
-    pub verifiers: Cid,
+    pub verifiers: Cid, // HAMT[Address]DataCap
     pub remove_data_cap_proposal_ids: Cid,
-    pub allocations: Cid, // HAMT[Address]HAMT[AllocationID]Allocation
+    pub allocations: Cid, // HAMT[ActorID]HAMT[AllocationID]Allocation
```

Comment:
While you're here something @arajasek mentioned earlier today is that he would find it helpful to see a comment around here that the ActorID for allocations HAMT is client and for claims HAMT is provider.